packages/bruno-app/src/components/CollectionSettings/Protobuf/index.js

```diff
@@ -0,0 +1,484 @@
+import React, { useState, useRef, useEffect } from 'react';
+import { useFormik } from 'formik';
+import { useDispatch } from 'react-redux';
+import StyledWrapper from './StyledWrapper';
+import toast from 'react-hot-toast';
+import { updateBrunoConfig } from 'providers/ReduxStore/slices/collections/actions';
+import cloneDeep from 'lodash/cloneDeep';
+import get from 'lodash/get';
+import { 
+  IconTrash, 
+  IconFile, 
+  IconFileImport, 
+  IconAlertCircle, 
+  IconFolder,
+  IconPlus
+} from '@tabler/icons';
+import { getRelativePath, getBasename, getDirPath } from 'utils/common/path';
+import { Tooltip } from 'react-tooltip';
+import { existsSync, resolvePath, browseDirectory, isDirectory } from '../../../utils/filesystem';
+
+const ProtobufSettings = ({ collection }) => {
+  const dispatch = useDispatch();
+  // Merge grpc and protobuf configs for backward compatibility
```

Comment:
@sanish-bruno This component should not know anything about backwards compatibility. This should be done at the electron layer.